addOns/spider/src/main/java/org/zaproxy/addon/spider/parser/SpiderHtmlParser.java

```diff
@@ -367,6 +373,30 @@ private boolean parseSource(HttpMessage message, Source source, int depth, Strin
             resourcesfound |= processAttributeElement(message, depth, baseURL, el, "background");
         }
 
+        // Process Blockquote elements
+        elements = source.getAllElements(HTMLElementName.BLOCKQUOTE);
+        for (Element el : elements) {
+            resourcesfound |= processAttributeElement(message, depth, baseURL, el, "cite");
+        }
+
+        // Process Param elements
+        elements = source.getAllElements(HTMLElementName.PARAM);
+        for (Element el : elements) {
+            resourcesfound |= processAttributeElement(message, depth, baseURL, el, "value");
+        }
```

Comment:
Would be good to keep them in alphabetic order even if some of them are not.

Reply:
Done.